### PR TITLE
Update unstructured logs

### PR DESCRIPTION
**Pull request by gaoguangze111**
Migrate to structured logs

file modified:
- staging/src/k8s.io/kube-aggregator/pkg/controllers/autoregister/autoregister_controller.go
- staging/src/k8s.io/kube-aggregator/pkg/controllers/cache.go
- staging/src/k8s.io/kube-aggregator/pkg/controllers/openapi/aggregator/aggregator.go

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Migrate unstructured logs to structured logs accoriding to https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md
**Which issue(s) this PR fixes**:

Fixes #
- staging/src/k8s.io/kube-aggregator/pkg/controllers/autoregister/autoregister_controller.go
- staging/src/k8s.io/kube-aggregator/pkg/controllers/cache.go
- staging/src/k8s.io/kube-aggregator/pkg/controllers/openapi/aggregator/aggregator.go
Ref:
- keps/sig-instrumentation/1602-structured-logging
- Structured Logging migration instructions
**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: staging/src/k8s.io/kube-aggregator/pkg/controllers/openapi/aggregator/aggregator.go [hunk 1]
```diff
@@ -106,10 +106,10 @@ func BuildAndRegisterAggregator(downloader *Downloader, delegationTarget server.
 	}
 
 	// Build initial spec to serve.
-	klog.V(2).Infof("Building initial OpenAPI spec")
+	klog.V(2).InfoS("Building initial OpenAPI spec")
 	defer func(start time.Time) {
 		duration := time.Since(start)
-		klog.V(2).Infof("Finished initial OpenAPI spec generation after %v", duration)
+		klog.V(2).InfoS("Finished initial OpenAPI spec generation after duration", "duration", duration)
```
Comment (lavalamp) [inline staging/src/k8s.io/kube-aggregator/pkg/controllers/openapi/aggregator/aggregator.go:112]:
```suggestion
		klog.V(2).InfoS("Finished initial OpenAPI spec generation", "duration", duration)
```

File: staging/src/k8s.io/kube-aggregator/pkg/controllers/openapi/aggregator/aggregator.go [hunk 2]
```diff
@@ -228,10 +228,10 @@ func (s *specAggregator) tryUpdatingServiceSpecs(specInfo *openAPISpecInfo) erro
 	if existedBefore && origSpecInfo != nil && origSpecInfo.etag == specInfo.etag {
 		return nil
 	}
-	klog.V(2).Infof("Updating OpenAPI spec because %s is updated", specInfo.apiService.Name)
+	klog.V(2).InfoS("Updating OpenAPI spec because specInfo is updated", "specInfo", specInfo.apiService.Name)
```
Comment (lavalamp) [inline staging/src/k8s.io/kube-aggregator/pkg/controllers/openapi/aggregator/aggregator.go:231]:
```suggestion
	klog.V(2).InfoS("Updating OpenAPI spec", "specInfo", specInfo.apiService.Name)
```

File: staging/src/k8s.io/kube-aggregator/pkg/controllers/openapi/aggregator/aggregator.go [hunk 3]
```diff
@@ -228,10 +228,10 @@ func (s *specAggregator) tryUpdatingServiceSpecs(specInfo *openAPISpecInfo) erro
...
 	defer func(start time.Time) {
 		duration := time.Since(start)
-		klog.V(2).Infof("Finished OpenAPI spec generation after %v", duration)
+		klog.V(2).InfoS("Finished OpenAPI spec generation after sometime", "duration", duration)
```
Comment (lavalamp) [inline staging/src/k8s.io/kube-aggregator/pkg/controllers/openapi/aggregator/aggregator.go:234]:
```suggestion
		klog.V(2).InfoS("Finished OpenAPI spec generation", "duration", duration)
```

File: staging/src/k8s.io/kube-aggregator/pkg/controllers/openapi/aggregator/aggregator.go [hunk 4]
```diff
@@ -260,10 +260,10 @@ func (s *specAggregator) tryDeleteServiceSpecs(apiServiceName string) error {
 		return nil
 	}
 	delete(s.openAPISpecs, apiServiceName)
-	klog.V(2).Infof("Updating OpenAPI spec because %s is removed", apiServiceName)
+	klog.V(2).InfoS("Updating OpenAPI spec because apiService is removed", "apiServiceName", apiServiceName)
 	defer func(start time.Time) {
 		duration := time.Since(start)
-		klog.V(2).Infof("Finished OpenAPI spec generation after %v", duration)
+		klog.V(2).InfoS("Finished OpenAPI spec generation after some time", "duration", duration)
```
Comment (lavalamp) [inline staging/src/k8s.io/kube-aggregator/pkg/controllers/openapi/aggregator/aggregator.go:266]:
```suggestion
		klog.V(2).InfoS("Finished OpenAPI spec generation", "duration", duration)
```

File: staging/src/k8s.io/kube-aggregator/pkg/controllers/cache.go [hunk 5]
```diff
@@ -29,13 +29,13 @@ import (
...
-	klog.Infof("Waiting for caches to sync for %s controller", controllerName)
+	klog.InfoS("Waiting for caches to sync for controller", "controller", controllerName)
 
 	if !cache.WaitForCacheSync(stopCh, cacheSyncs...) {
 		utilruntime.HandleError(fmt.Errorf("Unable to sync caches for %s controller", controllerName))
 		return false
 	}
 
-	klog.Infof("Caches are synced for %s controller", controllerName)
+	klog.Infof("Caches are synced for controller", "controller", controllerName)
```
Comment (yangjunmyfm192085) [inline staging/src/k8s.io/kube-aggregator/pkg/controllers/cache.go:39]:
```suggestion
	klog.InfoS("Caches are synced for controller", "controller", controllerName)
```

**Comment by yangjunmyfm192085** [comment]
@gaoguangze111 Could you rebase and modify it?

**Pull request closed without merging**